cfme/ansible_tower/explorer.py

```diff
@@ -89,14 +95,34 @@ def is_displayed(self):
 
 class TowerExplorerJobTemplatesAllView(TowerExplorerView):
     toolbar = View.nested(TowerExplorerSystemJobTemplatesToolbar)
+    entities = View.nested(BaseNonInteractiveEntitiesView)
```

Comment:
I think you should try `BaseEntitiesView`. entities are like provider, vm, host; it will give you `surf_page` and other facilities.